app/components/card-list.json

```diff
@@ -10,7 +10,8 @@
     "artist": "Ashlelang",
     "imageName": "Glueman.webp",
     "deckCardImage": "deckCard+Glueman.webp",
-    "cardNum": "1"
+    "cardNum": "1",
+    "collectionCount": 0
```

Comment:
the collectionCount is updated at runtime for the current user, correct? This could also potentially be optimized